### PR TITLE
Rename variables redefining builtins

### PR DESCRIPTION
**Pull request by Sophist-UK**

### CODE REVIEW

**Comment by samj1912** [comment]
This isn't entirely valid. We use "_" for gettext. Also you changed the "next" in the picard tag line in one of the files to "next-func"

File: picard/acoustid.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-# Picard, the next-generation MusicBrainz tagger
+# Picard, the next_func-generation MusicBrainz tagger
```
Comment (samj1912) [inline picard/acoustid.py]: Revert this

File: test/test_textencoding.py [hunk 2]
```diff
@@ -219,7 +219,7 @@ def test_incorrect(self):
     enclosed = u"⒐⒑⒒⒓⒔⒕⒖⒗⒘⒙⒚⒛⒜⒝⒞⒟⒠⒡⒢⒣⒤⒥⒦⒧⒨⒩⒪⒫⒬⒭⒮⒯⒰⒱⒲⒳⒴⒵ⒶⒷⒸⒹⒺⒻⒼⒽⒾⒿⓀⓁⓂⓃⓄⓅⓆⓇⓈⓉⓊⓋⓌⓍⓎⓏ" \
                 u"ⓐⓑⓒⓓⓔⓕⓖⓗⓘⓙⓚⓛⓜⓝⓞⓟⓠⓡⓢⓣⓤⓥⓦⓧⓨⓩ⓪⓫⓬⓭⓮⓯"
 
-    print("The following lines show the coverage of Latin characters conversion to ascii.")
+    print("The following lines show the coverage of Latin characters conversion to ascii_chars.")
```
Comment (samj1912) [inline test/test_textencoding.py]: Invalid change

**Comment by Sophist-UK** [comment]
I will rebase and put each file in a separate commit.

File: picard/util/thread.py [hunk 3]
```diff
@@ -36,10 +36,10 @@ def run(self):
 
 class Runnable(QRunnable):
 
-    def __init__(self, func, next):
+    def __init__(self, func, next_func):
         QRunnable.__init__(self)
         self.func = func
-        self.next = next
+        self.next = next_func
```
Comment (samj1912) [inline picard/util/thread.py]: I would prefer if the lhs variable is also self.next_func, just to maintain consistency

**Comment by Sophist-UK** [comment]
Done.

**Pull request merged**